src/style.css

```diff
@@ -297,7 +297,81 @@ body {
   border-radius: 0.5em;
   font-weight: 500;
   font-size: var(--fs-300);
-  line-height: 1.25em;
+  line-height: 3.25em;
 }
 
 /* Contact Form Styles */
+.contact-me {
+  background-image: url("assets/images/contact-form-bg.svg");
+  background-repeat: no-repeat;
+  background-position: 103% 30%;
+  background-size: 62% 100%;
+  background-color: var(--primary-color-400);
+  padding-top: 6em;
+  border-top-left-radius: 4em;
+  margin-top: -4em;
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+}
+
+.contact-me__title {
+  text-align: center;
+  color: var(--white);
+  font-size: var(--fs-900);
+  font-weight: 700;
+  line-height: 1.5em;
+}
+
+.contact-me__description {
+  max-width: 28em;
+  text-align: center;
+  padding: 0 1em;
+  color: var(--light-blue);
+  font-size: var(--fs-700);
+  line-height: 1.75em;
+}
+
+.contact-me__form {
+  margin-top: 2.25em;
+  max-width: 28em;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 2rem;
+}
+
+.contact-me__form__label {
+  display: none;
+}
+
+.contact-me__form__input {
+  width: 100%;
+  border-radius: 0.5em;
+  padding: 1em;
+  outline: none;
+  border: 2px solid transparent;
+}
+
+.contact-me__form__input:focus {
+  border-color: var(--black);
+}
+
+.contact-me__form__textarea {
+  min-height: 10em;
+  resize: vertical;
+  transition: 0.2s;
+}
+
+.contact-me__form__textarea:focus {
+  border-color: var(--black);
+}
+
+.contact-me__form__submit-button:hover:enabled {
+  background-color: #db574b;
+  color: var(--white);
+}
```

Comment:
- [ ] Kindly make sure to stick to the `Figma` design template as much as possible. Your button `hover` effect doesn't match with that of the `Figma` design so kindly make sure to fix it so your design looks exactly like the `Figma` design and more beautiful as well 😊. See detail for more details: 
<table>
<tr>
<th>Figma design</th>
<th>Your design</th>
</tr>
<tr>
<td>
<img width="789" alt="Screenshot 2023-05-02 at 20 29 25" src="https://user-images.githubusercontent.com/107192076/235779827-00447264-92db-4401-b794-0a9e30ef973a.png">

</td>
<td>
<img width="374" alt="Screenshot 2023-05-02 at 20 28 52" src="https://user-images.githubusercontent.com/107192076/235779878-477b6378-22ce-4a25-b184-d1bf025d46e8.png">

</td>
</tr>
</table>

Reply:
@Strangeal  The reason I changed the hover color is that it is the same as the background of the form, and it basically "disappears" when the user hovers over the button (the screenshot on the left).
In my implementation, that doesn't happen, and I thought that was better. 
 
![image](https://user-images.githubusercontent.com/110997325/235797080-af594c16-cfbb-493d-97b0-eba48a3c12f8.png)

Based on your request, I have reverted the commit that introduced those changes.